games/b.yaml

```diff
@@ -142,16 +142,25 @@
   url: http://cyxdown.free.fr/bs/
 
 - name: BitRiot
-  framework: SDL
   lang: C++
   license:
   - Apache
-  development: active
+  development: halted
   originals:
   - Bug Bomber
   repo: https://github.com/VenKamikaze/BitRiot
-  type: remake
-  updated: 2016-04-11
+  type: clone
+  updated: 2019-05-07
+  status: playable
+  multiplayer:
+  - Co-op
+  framework:
+  - SDL
```

Comment:
```suggestion
```
The game was ported to SDL2 so I don't think it supports SDL anymore

Reply:
Okay! The old version uses the SDL1. My bad!